db/schema.rb

```diff
@@ -232,40 +232,12 @@
     t.integer "void_date_value_check"
     t.integer "housingneeds_type"
     t.integer "housingneeds_other"
-    t.index ["created_by_id"], name: "index_case_logs_on_created_by_id"
```

Comment:
This looks like there's more going on here than what the migration does

Reply:
Yep, just fixing that, but something doesn't seem right with our schema.